cmd/root.go

```diff
@@ -178,6 +178,9 @@ func AddFlags(flagSet *pflag.FlagSet, cfg *config.Config) (configPath *string) {
 	flagSet.StringVar(&cfg.API.JSONListener, "grpc-json-listener",
 		cfg.API.JSONListener, "(Optional) endpoint to expose public grpc services via HTTP/JSON.")
 
+	flagSet.StringSliceVar(&cfg.API.JSONCorsAllowedOrigins, "grpc-cors-allowed-origins",
+		cfg.API.JSONCorsAllowedOrigins, "(Optional) coma delimited CORS Allowed Origins")
```

Comment:
```suggestion
		cfg.API.JSONCorsAllowedOrigins, "(Optional) comma delimited CORS Allowed Origins")
```

Reply:
I forgot that I made it a list, clarified that ;)